build-support/bin/generate_builtin_lockfiles.py

```diff
@@ -259,7 +259,7 @@ def generate_jvm_tool_lockfiles(tools: Sequence[JvmTool], dry_run: bool) -> None
 
 def generate(buildroot: str, tools: Sequence[Tool], args: Sequence[str], dry_run: bool) -> None:
     pants_repo_root = get_buildroot()
-    touch(os.path.join(buildroot, "BUILDROOT"))
+    touch(os.path.join(buildroot, "pants.toml"))
```

Comment:
The lack of `pants.toml` was leading to the native client emitting the error text:
```
Failed to read config file /tmp/tmp0cgz_lm3/pants.toml: No such file or directory (os error 2)
```

I also ran and discarded the successful lock update results of the following to test this:
```console
$ pants run build-support/bin/generate_builtin_lockfiles.py -- --all-python --all-jvm
```